utils/ipset.go

```diff
@@ -170,8 +170,9 @@ func (ipSet *IPSet) Create(setName string, createOptions ...string) (*Set, error
 	set := &Set{
 		Name:    setName,
 		Options: createOptions,
+		IPSet:   ipSet,
```

Comment:
map needs to be initialized before use: `ipSet.Sets = make(map[string]*Set)`

Reply:
Thanks for the review. Done it in NewIPSet function.

Reply:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x126c4fb]

goroutine 240 [running]:
github.com/cloudnativelabs/kube-router/utils.(*IPSet).run(0x0, 0xc42051b568, 0x2, 0x2, 0xc420450960, 0xc42062b128, 0x1bd963d, 0x6)
	/home/kube/go/src/github.com/cloudnativelabs/kube-router/utils/ipset.go:125 +0x9b
github.com/cloudnativelabs/kube-router/utils.(*Set).Flush(0xc4206cc690, 0xc420852d20, 0x19)
	/home/kube/go/src/github.com/cloudnativelabs/kube-router/utils/ipset.go:298 +0x7f
```

I ran into this panic, does the patch cover this as well?

Reply:
That panic *should* be fixed with this PR now. I am having success with the latest revision.